move/axelar_gateway/sources/gateway.move

```diff
@@ -276,6 +290,7 @@ public fun take_approved_message(
 
 fun version_control(): VersionControl {
     version_control::new(vector[
+        vector[],
```

Comment:
keep some functions available to test different cases

Reply:
cannot